lib/connectors/connector.ts

```diff
@@ -16,19 +22,31 @@ export interface Connector {
   _translator: Translator;
 
   /** Client that maintains an external database connection. */
-  _client: ConnectorClient;
+  _client?: ConnectorClient;
 
   /** Options to connect to an external instance. */
   _options: ConnectorOptions;
 
   /** Is the client connected to an external instance. */
-  _connected: boolean;
+  _connected?: boolean
```

Comment:
This is always defined:

```suggestion
  _connected: boolean
```